docs/security/rbac.md

```diff
@@ -366,6 +366,26 @@ You can create additional policies to further limit user access. Use the web UI
     ]
 }
 ```
+## Multiple Resources Statements
+
+Starting from version v1.54.0, LakeFS supports specifying multiple resources in a single RBAC statement.
```

Comment:
Also add the minimal fluffy version that is required with emphasis for on-prem enterprise users (in the cloud fluffy is managed for them)